src/components/settings-menu/DevNotificationsSection.tsx

```diff
@@ -78,8 +46,8 @@ const DevNotificationsSection = () => {
       [address]: { subscription: 'off', tx: state[address].tx },
     }));
 
-    firebaseUnsubscribeTopics('watcher', chainId, address);
-    firebaseUnsubscribeTopics('owner', chainId, address);
+    unsubscribeFromTransactionTopics('watcher', chainId, address);
+    unsubscribeFromTransactionTopics('owner', chainId, address);
```

Comment:
@pugson maybe we should block these settings on your upcoming PR when user is on a testnet, we'll be taking the testnet chain id which doesn't make sense here

Reply:
we'll also need to handle different chain ids when we support other chains

Reply:
yeah that whole screen is getting removed in my PR and mainnet is being hardcoded as chain id for now

Reply:
i can put in a small notice that says notifications are only working with mainnet if the user is on another chain

Reply:
nice! i think just removing the entry point is enough, we've been completely removing other features on testnets

Reply:
might be confusing ux when a menu disappears that was previously there. gonna ask design about that